rectools/models/vector.py

```diff
@@ -40,7 +41,8 @@ class VectorModel(ModelBase[ModelConfig_T]):
 
     u2i_dist: Distance = NotImplemented
     i2i_dist: Distance = NotImplemented
-    n_threads: int = 0  # TODO: decide how to pass it correctly for all models
+    recommend_n_threads: int = 0
```

Comment:
Since we accept these params in every vector model in init, why do we still need them here?

Reply:
Because we are accessing it in `VectorModel` methods?

Reply:
fixed